DM-2494 differentiate UI and backend change log for lwm2m use source … (#3180)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-10-18-510-0-improved-information-on-use-source-timestamp-configuration.md

ticket: DM-2494
version: 10.18.510.0
---

> The tooltip for the "Use source timestamp" in the LWM2M device configuration has been improved to explain how the LWM2M agent makes use of timestamp information if reported by a device.

Reviewer: @Oezge-Akin Also add "single" here as above?

Author: UI change is in LWM2M Configuration UI https://cumulocity.com/docs/device-integration/lwm2m/#lwm2m-configuration and not in single device registration.

> The tooltip for the "Use source timestamp" configuration in device's LWM2M Configuration view have been improved to explain how the LWM2M agent makes use of timestamp information if reported by a device.

Reviewer: ```suggestion
The tooltip for the "Use source timestamp" in the LWM2M device configuration has been improved to explain how the LWM2M agent makes use of timestamp information if reported by a device.
```
